### ISSUE TITLE
0 objects were processed during training

### ISSUE THREAD

**Issue by liuyu690806** [Microsoft/singleshotpose#158]
Hello, the author. When I was training, I had the following situation, the program in the loss.backward () quit directly without any error.At the same time, 0 objects were processed during training.
[图片](https://user-images.githubusercontent.com/64012147/108455853-faefef80-72a9-11eb-9ce7-f5a06ad4a70f.png)
[图片](https://user-images.githubusercontent.com/64012147/108455893-0d6a2900-72aa-11eb-9783-fb5c16355c60.png)
